app/views/content/research.md

```diff
@@ -19,3 +19,96 @@ The purpose of the survey was aimed at gaining further insights about those who
+</a>
+
+Published 20 May 2020
+
+## The Millennial Career Crossroads
+
+Kindred Agency, in partnership with SKV Communications, commissioned research company Censuswide to carry out a survey on behalf of the Get Into Teaching campaign, asking the full-time, part-time working public or job seekers (those aged between 21-40 years old and who studied in higher education) about their inclination and motivation for making significant changes to their lifestyle, including changing careers.
+
+<a href="/assets/documents/millenial-career-crossroads.pdf" class="call-to-action-icon-button">
+    Read the summary of findings (PDF 752KB)
+    <i class="fas fa-file-pdf"></i>
+</a>
+
+Published 7 January 2020
+
+## Exploring the career change conundrum
+
+Kindred Agency, in partnership with SKV Communications, commissioned research company Censuswide to carry out a survey on behalf of the Get Into Teaching campaign, asking the general public about their inclination and motivation for changing careers, specifically at the start of a new academic year.
+
+<a href="/assets/documents/career-conundrum.pdf" class="call-to-action-icon-button">
+    Read the summary of findings (PDF 460KB)
+    <i class="fas fa-file-pdf"></i>
+</a>
+
+Published 4 September 2019
+
+## Propensity for switching careers
+
+Kindred Agency, in partnership with SKV Communications, commissioned research company Censuswide to carry out a survey on behalf of the Get Into Teaching campaign, asking the general public about their propensity for changing career.
+
+<a href="/assets/documents/career-change-research.pdf" class="call-to-action-icon-button">
+    Read the summary of findings (PDF 450KB)
+    <i class="fas fa-file-pdf"></i>
+</a>
+
+Published 3 January 2019
+
+## Perceptions of influence survey
+
+Kindred Agency, in partnership with SKV Communications, commissioned research company Censuswide to carry out a survey asking the general public about the people who influence them, the significance of this influence and the influence and impact their teachers had on them. 
+
+<a href="/assets/documents/perceptions-of-influence.pdf" class="call-to-action-icon-button">
+    Read the summary of findings (PDF 450KB)
+    <i class="fas fa-file-pdf"></i>
+</a>
+
+Published 14 November 2018
+
+## Careers survey
+
+Censuswide were commissioned to ask university degree-level students and graduates from the last 18 months about their career aspirations. 
+
+<a href="/assets/documents/career-survey-summary.pdf" class="call-to-action-icon-button">
+    Read the summary of findings (PDF 424KB)
+    <i class="fas fa-file-pdf"></i>
+</a>
+
+Published 29 November 2017
+Last updated 6 April 2018
+
+## Soft messaging survey
+
+On behalf of NCTL, SKV Communications and Kindred Agency commissioned Censuswide to carry out a survey. This asked teachers and the non-teaching public from other professions about the extent to which they experience emotionally rewarding benefits from their career.
```

Comment:
Worth explaining who NCTL are or maybe linking to their homepage?

Reply:
Done